clients/imodelhub/config/rush-project.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
could you check that this json file wont be added to the actual pkg release?

i believe `npx pkgfiles` will list whats in the pkg we'd publish

Reply:
It gets added... is there a way to ignore besides manually editing each project's npmignore?

Reply:
unfortunately, not that im aware of. ngl, really annoying we need to add a specific file to each pkg to handle this :/ 
its what i didnt like about nx (tho ive never actually used it)

Reply:
![image](https://user-images.githubusercontent.com/1904889/186932893-0393cbc2-eeba-4e32-a530-dbd95f7912f4.png)
Could rush add those file at build time in the folders?

Reply:
Why is the config being used in the first place, its already ignored: 
![image](https://user-images.githubusercontent.com/1904889/186936404-14b52fea-d8f2-4130-a66e-911bd6e57e1a.png)

Reply:
Right... not sure... let me run pkgfiles again

Reply:
It's only included for a subset of the packages which do not have npmignore files (full-stack-tests, example-code\app for example)

Reply:
I guess the ones that do not already have `.npmignore` are not published anyway ?

Reply:
@aruniverse is this the case? If so, I'll remove the config completely from these projects.

Reply:
You can validate with rush.json project folder, when it says "shouldPublish". Strangely for the full-stack-tests, some are published, some are not. example-code is not published.

Reply:
None of the packages without .npmignore(s) are published, which makes sense, so I believe we're fine here